docs/arcaflow/workflows/expressions.md

```diff
@@ -473,7 +474,7 @@ Order (highest to lowest):
 
 ## Other information
 
-More information on the expression language is available in the [development guide](/arcaflow/contributing/expressions).
+More information on the expression language is available in the [development guide](/docs/arcaflow/contributing/expressions.md).
```

Comment:
Or `[development guide](../../contributing/expressions.md)`, which might be a bit more resistant to reorganization?

Reply:
FYI, the original link (`/arcaflow/contributing/expressions`) renders as `https://arcalot.io/arcaflow/contributing/expressions` on the [doc site](https://arcalot.io/arcaflow/workflows/expressions/#array-accessor), and that works.

Reply:
'development guide` sounds good to me

Reply:
Huh.  IIRC, both GitHub and my IDE objected to the lack of extension on the filename as well as the file path...so I'm not sure what to do, here.  I changed the path to relative per your suggestion, which seems like a good idea in general, and I've left the added extension...but, I'm wondering whether I shouldn't just revert this change and ignore GitHub and the IDE (although, I really hate ignoring warnings...because sometimes they are right...).